### PR TITLE
Compentency queries tests

### LINKED ISSUE #3: Write more test queries
* Input = location and or marker.  Output = cell types
   - All cells in loc X all cells of type Y
   - All cells expressing marker Z (more strictly, for which Z is recorded as a marker)
    - All cells in loc X of type Y expressing marker Z
* Input = cell types (and markers)
   - Location(s) of cell type X
   - Location(s) of cell type X expressing marker Z

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #3

### CODE REVIEW

File: tests/cells_by_location_marker.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+query_mechanism:
+  relationgraph:
+    query: "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+      PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+      PREFIX owl: <http://www.w3.org/2002/07/owl#>
+      PREFIX expresses: <http://purl.obolibrary.org/obo/RO_0015004>
+      PREFIX part_of: <http://purl.obolibrary.org/obo/BFO_0000050>
+      PREFIX cell: <http://purl.obolibrary.org/obo/CL_0000000>
+      PREFIX has_characterizing_marker_set: <http://purl.obolibrary.org/obo/RO_0015004>
```
Comment (dosumis) [inline tests/cells_by_location_marker.yaml]: why do we need this + expresses when we can infer expresses from the property chain we just added?

Comment (anitacaron) [reply tests/cells_by_location_marker.yaml]: Ops forgot to remove it.

Comment (dosumis) [reply tests/cells_by_location_marker.yaml]: THought you might have as see 'expresses' used elsewhere. 

BTW - what do the reports look like or do these still need work?  Couldn't spot anything in GA readout.   Might be best for users if we could save a reporting log to repo.

Comment (anitacaron) [reply tests/cells_by_location_marker.yaml]: The output is like this:

Screenshot 2023-01-30 at 13 09 45

I can put that in a file.

**Pull request merged**